Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -178,13 +178,34 @@ void OpenALStream::SoundLoop()
   surround_capable = false;
 #endif
 
-  u32 ulFrequency = m_mixer->GetSampleRate();
-  numBuffers = SConfig::GetInstance().iLatency + 2;  // OpenAL requires a minimum of two buffers
+  u32 ul_frequency = m_mixer->GetSampleRate();
+  // OpenAL requires a minimum of two buffers
+  num_buffers = SConfig::GetInstance().iNumBuffers;
```

Comment:
I'm still against the use of two user configuration variables when one would do fine. Can you peg num_buffers at 2 or 3?